dashboard/app/helpers/plc/courses_helper.rb

```diff
@@ -1,5 +1,5 @@
 module Plc::CoursesHelper
   def options_for_user_enrollment_courses
-    Plc::Course.pluck(:name, :id).sort
+    Plc::Course.all.map {|c| [c.name, c.id]}.sort
```

Comment:
These are equivalent:
```
irb(main):010:0> Plc::Course.all.map{|c| [c.name, c.id]}.sort == Plc::Course.pluck(:name, :id).sort
=> true
```

Why the change? I find the former (slightly) more readable